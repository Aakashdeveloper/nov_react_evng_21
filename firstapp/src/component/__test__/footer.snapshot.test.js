import React from 'react';
import Footer from '../footer';
import {create} from 'react-test-renderer';

describe('Snapshot test',() => {
    test('Testing Footer',() => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})