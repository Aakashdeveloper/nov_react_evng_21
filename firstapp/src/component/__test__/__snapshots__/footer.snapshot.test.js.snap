// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Testing Footer 1`] = `
<div
  style={
    Object {
      "display": "inline-block",
      "width": "100%",
    }
  }
>
  <hr />
  <center>
    <h3>
      Â© Developers Funnel 
       
    </h3>
  </center>
</div>
`;
